examples/models/llama/export_llama_lib.py

```diff
@@ -588,25 +588,9 @@ def _prepare_for_llama_export(args) -> LLMEdgeManager:
     )
 
     # At this point, the model is loaded in the default fp32.
-
-    # Checkpoint dtype should be lower or equal precision to the dtype override.
+    # override dtype
     checkpoint_dtype = edge_manager.model.checkpoint_dtype
```

Comment:
Not sure why we need to keep this original checkpoint_dtype, since it would be overriden in the line below. Just to keep the behavior of the existing code. @jackzhxng and @kimishpatel , what's the reason behind this, for weight quantization?